roles/swapfile/tasks/main.yml

```diff
@@ -7,7 +7,7 @@
   register: swapfile_check
 
 - name: create swap file {{ swapfile_path }}
-  command: fallocate -l {{ swapfile_size }} {{ swapfile_path }}
+  command: dd if=/dev/zero of={{ swapfile_path }} count={{ swapfile_size }} bs=1MiB
```

Comment:
This is failing with `[E303] Use ":" YAML format when arguments are over 3`, i've tried:
```
  command: 
     dd if=/dev/zero of={{ swapfile_path }} count={{ swapfile_size }} bs=1MiB

```

```
  command: 
     "dd if=/dev/zero of={{ swapfile_path }} count={{ swapfile_size }} bs=1MiB"
```

```
  command: "dd if=/dev/zero of={{ swapfile_path }} count={{ swapfile_size }} bs=1MiB"
```

all seem to fail with the same error

Reply:
This is a result of the dd syntax. It's the same as yaml syntax and is confusing the linter. I suggest disabling that lint rule for this task.

Reply:
You need to use `warn: False` to skip ansible-lint for this task.

Reply:
I'm guessing this is a bug in ansible-lint, but i've even tried skipping it via: https://github.com/willthames/ansible-lint#false-positives   and i can't get it to pass 😞